packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -268,6 +271,33 @@ class EnrollVerbHandler extends AbstractVerbHandler {
         'private:${enrollParams.otp?.toLowerCase()}${AtSecondaryServerImpl.getInstance().currentAtSign}');
   }
 
+  // Store enrollment info after approval in a public hidden key
+  Future<void> _storeEnrollmentInPublicHiddenKey(
+      String newEnrollmentId,
+      EnrollDataStoreValue enrollmentDataStoreValue,
+      String currentAtSign) async {
+    var key = 'public:$enrollmentPublicHiddenKey';
```

Comment:
Wait does this actually work as intended? A public key with `_` is also hidden? I was under the impression that this was only for self-keys, but it makes sense that it works for all key types.

If so, the changes I proposed in https://github.com/atsign-foundation/at_protocol/pull/166 are not quite right.